tests/pytests/functional/modules/state/requisites/test_require.py

```diff
@@ -358,8 +357,10 @@ def test_requisites_require_ordering_and_errors_3(state, state_tree):
         - name: echo A first
     """
     errmsg = (
-        'Cannot extend state foobar for ID A in "base:requisite". It is not part of the'
-        " high state."
+        "Cannot extend ID 'A' in 'base:requisite'. It is not part of the high state.\n"
+        "This is likely due to a missing include statement or an incorrectly typed ID.\n"
+        "Ensure that a state with an ID of 'A' is available\n"
+        "in environment 'base' and to SLS 'requisite'"
```

Comment:
shouldn't the name of the environment be 'dynamic', taking it from `saltenv` if available? It can be different than `base`

Reply:
No it shouldn't be. no `saltenv` is being passed to the test. so it will be default env. and that would be `base`. if the test comes back with a different saltenv then what is setup in the test then that would be a failure. envs are tested elsewhere so no need to test dynamic saltenvs here.